benchmarks/ctia/tasks/migrate_es_stores_bench.clj

```diff
@@ -62,18 +62,17 @@
 (defgoal migrate-store-indexes "Run a migration"
   :setup (fn [& args]
            (println "launching CTIA...")
-           (setup-ctia-es-store!)
+           (let [{:keys [app]} (setup-ctia-es-store!)]
+             (println "clear ES stores")
+             (delete-store-indexes)
 
-           (println "clear ES stores")
-           (delete-store-indexes)
-
-           (println "posting fixtures...")
-           (post-fixtures)
-           (prn "fixtures loaded!")
-           [true])
-  :cleanup (fn [& args]
+             (println "posting fixtures...")
+             (post-fixtures)
+             (prn "fixtures loaded!")
+             [app]))
+  :cleanup (fn [app]
              (println "cleanup and exit CTIA...")
-             (cleanup-ctia!)))
+             (cleanup-ctia! app)))
 
 (defn run-migrate-es-stores []
```

Comment:
I will update this benchmark to make it comply to the migration changes: https://github.com/threatgrid/iroh/issues/4231

Reply:
Thanks!